### DISCUSSION TITLE
Support for Swift Serverless Functions (Official Swift AWS Lambda Runtime now available!)

### DISCUSSION THREAD

**Discussion by jeffreyrobert** [vercel/vercel#4746] [Ideas] [answered]
Hey guys, absolutely love your product!

I know this request has been made before but thought it would be worth revisiting since the official [Swift AWS Lambda Runtime](https://swift.org/blog/aws-lambda-runtime/) is now available.

[Apple released a great WWDC video](https://developer.apple.com/videos/play/wwdc2020/10644/) about development, testing, and deployment workflows.

Your deployment workflow on Vercel is just fantastic, I would love to be able to use it to deploy Serverless Functions written in Swift. I know there are probably many requests for many languages. The Swift community is growing extremely quickly and Swift is a perfect fit for Serverless Functions due to it's light resource usage.

Let me know if this is anywhere on your radar. Keep up the amazing work. Thank you!

**Comment by nonfungibletunji** [comment]
+1

**Comment by styfle** [comment, marked as answer]
Hi @jeffreyrobert 

We don't have plans to add new languages at this time.

However, you can create your own [Custom Runtime](https://vercel.com/docs/runtimes#advanced-usage/community-runtimes) if you wish to support a different language.

Then anyone can configure the [`functions`](https://vercel.com/docs/configuration#project/functions) property in `vercel.json` to use your Custom Runtime.

Read more about [Developing a Runtime](https://github.com/vercel/vercel/blob/master/DEVELOPING_A_RUNTIME.md).
